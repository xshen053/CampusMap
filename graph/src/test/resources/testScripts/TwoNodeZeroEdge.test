# Tests adding two nodes to the graph

# Create a graph
CreateGraph TNZE_graph

# Add two nodes
AddNode TNZE_graph a
AddNode TNZE_graph b

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
ListNodes TNZE_graph
ListChildren TNZE_graph a
ListChildren TNZE_graph b