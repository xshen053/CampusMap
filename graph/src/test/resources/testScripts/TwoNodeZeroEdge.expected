# Tests adding two nodes to the graph

# Create a graph
created graph TNZE_graph

# Add two nodes
added node a to TNZE_graph
added node b to TNZE_graph

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
TNZE_graph contains: a b
the children of a in TNZE_graph are:
the children of b in TNZE_graph are: